frontends/p4/p4-parse.ypp

```diff
@@ -813,6 +827,34 @@ actionRef
                                            $$ = new IR::ActionListElement(@2, $1, mce); }
     ;
 
+entry
+    : optAnnotations keysetExpression ':' actionInvocation
```

Comment:
@mbudiu-vmw: there is no definition for default_action, it falls on the generic rule
```
    | optAnnotations optCONST IDENTIFIER '=' initializer ';'
```
and initializer is in fact an expression.
In fact reviewing the grammar to look for an action invocation, the only pertinent rule I found is ```assignmentOrMethodCallStatement```, and I borrowed the rules for actionInvocation from there. I think that a more clean implementation should split that rule into the assignment and actionInvocation and in that case there are no additional rules, and the intent is much more obvious.